src/main/kotlin/nl/avisi/kotlinwebtest/jdbc/dsl/Dsl.kt

```diff
@@ -0,0 +1,66 @@
+/**
+ * For licensing, see LICENSE.txt
+ * @author Martijn Heuvelink
+ */
+package nl.avisi.kotlinwebtest.jdbc.dsl
+
+import nl.avisi.kotlinwebtest.StepBuilder
+import nl.avisi.kotlinwebtest.TestConfiguration
+import nl.avisi.kotlinwebtest.WebTest
+import nl.avisi.kotlinwebtest.expressions.ConstantExpression
+import nl.avisi.kotlinwebtest.expressions.Expression
+import nl.avisi.kotlinwebtest.jdbc.BooleanValidator
+import nl.avisi.kotlinwebtest.jdbc.JdbcRequestDefaults
+import nl.avisi.kotlinwebtest.jdbc.JdbcTestConfiguration
+import nl.avisi.kotlinwebtest.jdbc.JdbcTestStep
+import nl.avisi.kotlinwebtest.jdbc.QueryValidator
+import nl.avisi.kotlinwebtest.jdbc.RowCountValidator
+
+class JdbcSettingsBuilder(val configuration: TestConfiguration) {
+    fun default(init: DefaultSettingsBuilder.() -> Unit) {
+        val builder = DefaultSettingsBuilder(configuration[JdbcTestConfiguration::class].defaults)
+        builder.init()
+    }
+
+    inner class DefaultSettingsBuilder(val request: JdbcRequestDefaults)
+}
+
+infix fun JdbcTestStep.validate(init: Validation.() -> Unit) {
+    Validation(this).init()
+}
+
+class Validation(private val step: JdbcTestStep) {
+    fun rowCount(amount: Int) = step.validators.add(RowCountValidator(amount))
```

Comment:
I believe we use row_count as naming convention (not standard, I know, but it reads better when used as a DSL)